src/styles/global.less

```diff
@@ -143,6 +143,7 @@ align-items: center;
     padding: 30px 0px 30px 0px;
     z-index: 999;
     position: absolute;
+    
```

Comment:
Mind removing this extra line?

Reply:
will get to all of these! looks like these were from previous cohort but I will clean them up